setup-rtd.rst

```diff
@@ -1,6 +1,5 @@
 .. _setup-rtd:
 
-=================================================
```

Comment:
I thought we were using the overline for H1?

Reply:
After discussion with @evildmp , we decided to leave them out.
His take is that when you have the title with overline, the logical next heading level would be `=` without overline. And imo it's better to keep this style guide in sync with the OpenStack one except for the overlines than prescribing completely different characters for the heading levels.